### PR TITLE
[14.0][MIG] base_maintenance

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration

### CODE REVIEW

**Review by etobella — APPROVED** [review]

**Review by lmiguens-solvos — APPROVED** [review]
LGTM

File: base_maintenance/views/maintenance_team_views.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                <group>
+                    <field
+                        name="description"
+                        placeholder="Description of the team's function"
+                    />
+                </group>
+            </xpath>
+        </field>
+    </record>
+    <!-- There is no search view to inherit, so creating it-->
```
Comment (jarroyomorales) [inline base_maintenance/views/maintenance_team_views.xml]: This search view is already added in Odoo's core so its not needed here. I just realized it was like this also in v13 and v12, so maybe we should do a PR fixing it.

File: base_maintenance/views/report_maintenance_request.xml [hunk 2]
```diff
@@ -106,14 +106,19 @@
...
-        <report
-            id="action_report_maintenance_request"
-            string="Maintenance Request"
-            model="maintenance.request"
-            report_type="qweb-pdf"
-            file="base_maintenance.report_maintenance_request"
-            name="base_maintenance.report_maintenance_request"
-            print_report_name="'Maintenance Request - %s' % (object.name)"
-        />
+        <record id="action_report_maintenance_request" model="ir.actions.report">
```
Comment (jarroyomorales) [inline base_maintenance/views/report_maintenance_request.xml:109]: The report doesnt appear in the request's form

**Review by jarroyomorales — APPROVED** [review]

File: base_maintenance/views/maintenance_team_views.xml (hunk 1 above)
Comment (dalonsod) [reply base_maintenance/views/maintenance_team_views.xml]: Done!

File: base_maintenance/views/report_maintenance_request.xml (hunk 2 above)
Comment (dalonsod) [reply base_maintenance/views/report_maintenance_request.xml:109]: @jarroyomorales  I think the problem is that report record needs `binding_type` and `binding_model_id` fields. The first one is already fixed, but the second one not yet. On runbot I've clicked on "Add in the print menu" button to achieve it.

In my local environment adding this line in order to permanently fix it:

`<field name="binding_model_id" ref="model_maintenance_request" />`

causes a parse error. And I didn't solve it yet... :(

Comment (jarroyomorales) [reply base_maintenance/views/report_maintenance_request.xml:109]: Oh, I tested on runbot and saw the button, thats why I approved the PR. 
Maybe `ref="maintenance.model_maintenance_request"` solves it?

Comment (dalonsod) [reply base_maintenance/views/report_maintenance_request.xml:109]: Thank you so much! :D

**Pull request merged**